articles/quickstart/native/react-native/index.yml

```diff
@@ -32,7 +32,7 @@ github:
   repo: auth0-react-native-sample
 sdk:
   name: react-native-auth0
-  url: https://github.com/auth0/react-native-auth0
+  url: https://github.com/auth0/react-native-auth0/tree/vnext
```

Comment:
Should this point to the `vnext` branch?

Reply:
Nice catch, updated links